.github/workflows/dependabot.yml

```diff
@@ -54,28 +54,27 @@ jobs:
           ssh-add -D
           ssh-agent -k
 
-      - name: Clear no-test label
+      - name: Clear [no-test] prefix from PR title
+        if: ${{ contains(github.event.pull_request.title, '[no-test]') }}
         uses: actions/github-script@v6
+        env:
+          TITLE: '${{ github.event.pull_request.title }}'
         with:
           script: |
-            try {
-              await github.rest.issues.removeLabel({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                issue_number: context.issue.number,
-                name: 'no-test'
-              });
-            } catch (e) {
-              if (e.name == 'HttpError' && e.status == 404) {
-                /* expected: 404 if label is unset */
-              } else {
-                throw e;
-              }
-            }
+            const title = process.env['TITLE'].replace(/\[no-test\]\W+ /, '')
```

Comment:
Why the `env` dance? This could just directly use `github.event.pull_request.title` surely?

Reply:
Uh, because I have no confidence in my scripting of github actions, it's frustrating to test this seems to be the most easy to get to work solution without magic. This example claims your suggestion works however https://github.com/actions/github-script#use-esm-import

Reply:
Not sure how this is related to imports -- your script already uses the `github` object, it's just about dropping that title env and replacing it with
```js
const title = github.event.pull_request.title.replace(/\[no-test\]\W+ /, '')
```

Reply:
IF only it was that easy https://github.com/jelly/cockpit-podman/actions/runs/6348462088/job/17245272304?pr=32

The example in the github-script page makes it even more of a soup:

```
            const { default: printStuff } = await import('${{ github.workspace }}/src/print-stuff.js')
```
So it seems they do some templating and then run the nodejs script.

Reply:
If you don't mind we keep it this way.

Reply:
Weird.. ok.